### ISSUE TITLE
latest prebuilt binary are missing several tabs in Settings

### ISSUE THREAD

**Issue by jwcasl** [huo-ju/photoprism-freebsd-port#53]
Thank you for the port and the prebuilt binaries. The latest binary (11-2022) seems to be missing three tabs in Settings: Advanced, Services, and Account. Reverting back to the 6-2022 binary would bring these tabs back. Thanks for looking into this.

**Comment by Henry78** [comment]
Not missing here!? I remember I had to wait quite a while after upgrading until everything worked (i.e. also the favorite/heart icon for images was missing).

**Comment by jwcasl** [comment]
Thanks for replying and it's good to know that it works for others so it's probably an issue on my end. I have waited now more than a month but they are still missing. I've tried reinstalling with and without erasing the entire assets folder to prevent issues from prior installations but still don't see these three tabs. This is in a FreeBSD 12.3 jail created using bastille that had successfully run prior versions of Photoprism without issues. I followed this guide: https://www.truenas.com/community/threads/install-guide-photoprism-in-a-jail-on-truenas-core-13-0.106992/ which is helpful to add MariaDB to help speed up the database backend, though I don't run truenas. The only thing I've not tried is to make a new jail and just install from scratch, which I think I might have to do now.
